ajax/libs/libsodium-wrappers/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "libsodium-wrappers",
+  "version": "0.2.10",
+  "filename": "modules/libsodium-wrappers.min.js",
```

Comment:
in [dist/modules](https://github.com/jedisct1/libsodium.js/tree/0.2.10/dist/modules) i didnt find libsodium-wrappers.min.js, maybe you should check it again :p